solidity/depool/README.md

```diff
@@ -172,14 +172,14 @@ tvm_linker decode --tvc DePoolProxy.tvc
 
 `"participantRewardFraction":*number*` - percentage of the total DePool reward (in integers, up to 99 inclusive) that goes to Participants.
```

Comment:
May be we should also specify the recommended value (95)

Reply:
added